spec.md

```diff
@@ -1,81 +1,401 @@
+platforms to add support. Furthermore, the libraries themselves for validating
+and signing will be WASM compatible allowing for embedding within websites.
+Furthermore, the use of a subtitles file makes this specification format-agnostic
+although the libraries will be written to only support AV1 to make it easier
+for testing.
+
+However, it is noted, that platforms will probably not perform the verification
+on device, instead verifying it on their servers and showing that the
+verification passed to the users.
+
 ### Cost analysis
 
+The extra cost burden will mostly be on individual users having to verify the
+hashes generated, as well as the users generating the signatures in the first
+place.
+
+Due to the storage process only requires one extra subtitle format, the
+increased cost to the distribution platforms will be negligible.
+
+<!-- TODO:
+
+Calculate how large the subtitle files will be to verify the above statement
+
+-->
+
 ### Security considerations
 
+Due to this project detailing how to validate authorisation of an individual,
+security is at the heart of considerations. Therefore, all aspects of the
+designed specification has been thoroughly reviewed about how it could
+potentially be abused to manipulate end users into a false sense of trust.
+
+For example, chunks can be expandable, this allows context to be included in
+a signature requiring embeddings to have all information if they wish to be
+verifiable. Chunks do have a limit to the length they can be, to make balance
+the essence of streaming, but this still can be used to the signers advantage,
+by ending the chunk mid sentence or mid word which would make it obvious to the
+end user that there was more information to gleam. But this has the
+disadvantage that it requires a significant amount of input by the signer,
+which could cause complications during a livestream. Work arounds do exist, for
+example having software that ends a chunk when the signer or trusted third
+party presses a button and starts a new one.
+
+Furthermore, a minimum chunk size is specified to help reduce the chance of
+splicing, however full mitigations will be left for the signer to implement,
+for example ending a chunk mid word. Although, to balance ease of use, it is
+also recommended that a simple set chunk size could be used to prevent most
+circumstances.
+
+<!-- TODO:
+
+- making fake credential information to appear as if it was official
+
+-->
+
 ### Privacy concerns
 
+Due to the lack of data stored and all information shared is approved by the
+signer themselves, there is little concern for privacy.
+
+But it should be made known to the signers that any information included in
+their credentials will be viewable to all users, which includes their email.
+
+It should also be known that there is a potential for a user been tracked
+online by their signatures if they use the same public/private key pair for
+two different accounts.
+
```

Comment:
Definitely a bit short and lack luster for privacy concerns, see https://www.w3.org/TR/vc-data-model/#privacy-considerations